build.py

```diff
@@ -91,6 +91,20 @@ def test(self, compiler_name, compiler_version, image_name):
                                   "-s compiler.libcxx=libstdc++ --build" %
                                   (container_name, compiler_name, compiler_version), shell=True)
 
+            subprocess.check_call("docker exec %s conan remote add conan-community "
+                                  "https://api.bintray.com/conan/conan-community/conan "
+                                  "--insert" %
+                                  container_name, shell=True)
+
+            subprocess.check_call("docker exec %s conan install gtest/1.8.0@conan/stable -s "
+                                  "arch=x86_64 -s compiler=%s -s compiler.version=%s "
+                                  "-s compiler.libcxx=libstdc++ --build" %
+                                  (container_name, compiler_name, compiler_version), shell=True)
+            subprocess.check_call("docker exec %s conan install gtest/1.8.0@conan/stable "
+                                  "-s arch=x86 -s compiler=%s -s compiler.version=%s "
+                                  "-s compiler.libcxx=libstdc++ --build" %
+                                  (container_name, compiler_name, compiler_version), shell=True)
+
             if compiler_name == "clang":
```

Comment:
@lasote I didn't understand why clang builds are replicated at this point.

Reply:
? Neither do I, maybe we can remove it?

Reply:
Done!